editor/src/components/inspector/common/control-status.ts

```diff
@@ -174,7 +176,9 @@ function isControlled(
       return spiedValue != null
     } else if (
       modifiableAttributeIsAttributeValue(modifiableAttributeResult.value) ||
-      modifiableAttributeIsPartOfAttributeValue(modifiableAttributeResult.value)
+      modifiableAttributeIsPartOfAttributeValue(modifiableAttributeResult.value) ||
+      modifiableAttributeIsAttributeNestedArray(modifiableAttributeResult.value) ||
+      modifiableAttributeIsAttributeNestedObject(modifiableAttributeResult.value)
```

Comment:
array literals and object literals are not considered controlled, because they are expandable without clearing their existing content